tests/e2e/scale/noobaa/test_warp.py

```diff
@@ -38,9 +39,83 @@ class TestWarp(E2ETest):
     to ensure that noobaa pods are still in a running state
     """
 
+    GCP_ACCOUNT_SKIP = "GCP Account Credentials Issue"
+
     @bugzilla("2089630")
     @pytest.mark.polarion_id("OCS-4001")
-    def test_s3_benchmark_warp(self, warps3, mcg_obj, bucket_factory):
+    @pytest.mark.parametrize(
+        argnames="amount,interface,bucketclass_dict",
+        argvalues=[
+            pytest.param(
+                *[1, "OC", None],
+            ),
+            pytest.param(
+                *[
+                    1,
+                    "OC",
+                    {
+                        "interface": "OC",
+                        "backingstore_dict": {
+                            "pv": [(1, 100, DEFAULT_STORAGECLASS_RBD)]
+                        },
+                    },
+                ],
+            ),
+            pytest.param(
+                *[
+                    1,
+                    "OC",
+                    {
+                        "interface": "OC",
+                        "backingstore_dict": {"aws": [(1, "eu-central-1")]},
+                    },
+                ],
+            ),
+            pytest.param(
+                *[
+                    1,
+                    "OC",
+                    {"interface": "OC", "backingstore_dict": {"azure": [(1, None)]}},
+                ],
+            ),
+            pytest.param(
+                *[
+                    1,
+                    "OC",
+                    {"interface": "OC", "backingstore_dict": {"gcp": [(1, None)]}},
+                ],
+                marks=[
+                    pytest.mark.skip(GCP_ACCOUNT_SKIP),
+                ],
```

Comment:
If this is here because of the missing / obsolete GCP credentials at auth.yaml?

If that's the case, then we can and should remove this mark because `bucket_factory` will skip the test when the ULS creation will fail. This way the test will resume running once the credentials are fixed without the need of an additional PR.